assets/component-menu-drawer.css

```
@@ -215,11 +215,27 @@ details[open].menu-opening > .menu-drawer__submenu {
 }
 
 .menu-drawer__utility-links {
-  padding: 2rem;
+  padding: 0;
   background-color: rgba(var(--color-foreground), 0.03);
   position: relative;
 }
 
+.menu-drawer__utility-links--social-media-links {
+  padding: 2rem;
+}
+
+@media screen and (max-width: 749px) {
+  .menu-drawer__utility-links--account {
```

Probably too much just to prevent the styling from being applied multiple times. 😅 
```suggestion
  .menu-drawer__utility-links--account:where(:not(.menu-drawer__utility-links--social-media-links):not(.menu-drawer__utility-links--localizations)) {
```

---

Honestly, since there is no harm I'd keep the classes simple and more readable 🙂. At the same time, I don't strong feeling about it. I'm okay to add what you suggest if @Ken votes for this too 🙂.

---

I guess I kind of like this type of thing being handled via classes a lot of the time. Not necessarily ridiculous BEM monstrosities like `menu-drawer__utility-links--social-media-links` 😅  but generally. 

To poke at this in another way though (sorry), the bug we're fixing right now is specific to the drawer utility links element, but the conditions we're checking are potentially applicable to anything in the header. Would it make sense to add classes like `header--has-account`, `header--has-localizations`, and `header--has-social` at a higher level so we can potentially make use of them in the future without having to rewrite all this code? I've previously had to (or wanted to) refactor a lot modifier classes and liquid conditionals done too deep in the tree.

---

It's wise. Thanks Ken for a good idea. 

It's done ✅